pkg/nodeinfo/attributes.go

```diff
@@ -37,6 +37,7 @@ const (
 	NodeLabelNvGPU            = "nvidia.com/gpu.present"
 	NodeLabelWaitOFED         = "network.nvidia.com/operator.mofed.wait"
 	NodeLabelCudaVersionMajor = "nvidia.com/cuda.driver.major"
+	NodeLabelOSTreeVersion    = "feature.node.kubernetes.io/system-os_release.OSTREE_VERSION"
```

Comment:
nit: bundle NFD labels together ? :)

not really a must just if you feel like it.